components/scripts/gradle/03-validate-local-build-caching-different-locations.sh

```diff
@@ -38,6 +38,10 @@ ge_server=''
 interactive_mode=''
 
 main() {
+  if [[ "$_arg_debug" == "on" && "$offline_mode" == "on" ]]; then
+    debug "Running experiment in offline mode."
+  fi
+
```

Comment:
I'm not sure about the relationship established here between debug and offline.

Reply:
This could be simplified by removing the debug check. The debug function does that check already.

Reply:
Removed the redundant debug check in 77a18f5527cc5fba4c16e59daa112563a5e74bd8.